### PR TITLE
new /desktop/organisations page - 18.04

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Build new organisation page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/organisations>
- Check page matches [the copy doc](https://docs.google.com/document/d/1q3_fQB3PUGbREsbCC4FnmHrpgn9nIWtMwyhDC6CXhNE/edit#) and [the design](https://user-images.githubusercontent.com/36884067/38875497-65deee80-4252-11e8-997d-01adb6b310b3.jpg)

## Issue / Card

Fixes [#470](https://github.com/ubuntudesign/web-squad/issues/470)

### CODE REVIEW

File: templates/desktop/organisations.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+      <p>Ubuntu Advantage is the professional support package from the experts at Canonical. Get 24-hour phone and web support every day with access to engineers with first-hand experience of your issues. It also includes Landscape, the Ubuntu systems management tool, for monitoring, managing, patching and compliance reporting on all your Ubuntu desktops.</p>
+      <p><a href="/support">Learn about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}4125e046-landscape_activity.png" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
```
Comment (pmahnke) [inline templates/desktop/organisations.html]: can you update with this image? https://assets.ubuntu.com/v1/d8c957cc-2018-04-20-release-eof-desktop.png  also, it should be on the left from the design

File: navigation.yaml [hunk 2]
```diff
@@ -109,7 +109,7 @@ desktop:
     - title: For partners
       path: /desktop/partners
     - title: Ubuntu in organisations
-      path: /desktop/ubuntu-in-organisations
+      path: /desktop/organisations
```
Comment (pmahnke) [inline navigation.yaml]: @frankiegranato I don't think this scans well

image

could it be 'In organisations' or 'For organisations' ?

File: templates/desktop/organisations.html [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+{% block title %}Ubuntu in organisations{% endblock %}
+
+{% block content %}
+
+<section class="p-strip is-deep" style="background-image: url({{ ASSET_SERVER_URL }}214648ba-_M8A2368-final.jpg?w=1280); background-repeat: no-repeat; background-size: cover; background-position: bottom center;">
+  <div class="row">
+    <div class="col-12 suffix-7">
+      <div class="p-card--overlay">
+        <h1>Ubuntu in organisations</h1>
+        <p>Ubuntu is adopted by small and large teams alike because it is easy to use, highly secure, has a low cost of ownership and has a huge range of apps.</p>
```
Comment (pmahnke) [inline templates/desktop/organisations.html]: remove second 'has' as in  'and a huge range of apps'

File: navigation.yaml (hunk 2 above)
Comment (steverydz) [reply navigation.yaml]: I went with 'For...' to match the others

**Comment by steverydz** [comment]
@pmahnke Changes addressed

**Comment by steverydz** [comment]
@frankiegranato Here is where you can review this: http://www.ubuntu.com-pr-2958.run.demo.haus/desktop/organisations

**Comment by frankiegranato** [comment]
A couple of v minor amends in the copydoc (sorry @steverydz).

**Pull request merged**